dashboard/src/app/(authenticated)/(diagram)/components/ThreadPanel.tsx

```diff
@@ -1,10 +1,10 @@
-import { WfRun, WfSpec } from 'littlehorse-client/proto'
-import { ChevronLeftIcon, ChevronRightIcon } from 'lucide-react'
-import { useRouter } from 'next/navigation'
-import { FC, useMemo } from 'react'
-import { useReplaceQueryValue } from '../hooks/useReplaceQueryValue'
-import { useScrollbar } from '../hooks/useScrollbar'
-import { useThread } from '../hooks/useThread'
+import { WfRun, WfSpec } from 'littlehorse-client/proto';
+import { ChevronLeftIcon, ChevronRightIcon } from 'lucide-react';
+import { useRouter } from 'next/navigation';
```

Comment:
I'm curious which linting rules are you using, this is adding semicolons but we don't do that.

Please make sure to run `npm run lint:fix`